eventing/sources/Sources.md

```diff
@@ -0,0 +1,44 @@
+# Knative Event Sources
+
+This is a non-exhaustive list of Event sources for Knative. Inclusion in this list **does not**
+imply an endorsement, nor does it imply any level of support.
+
+This is a generated file and should not be changed manually. All changes should follow the procedure
```

Comment:
You should be able to add a comment in `<!-- this markdown -->`

Reply:
Left it visible at the top of the document. Do you have a preference for it being a comment?

Reply:
Made it a comment.